charts/rasa-x/templates/rasa-secret.yaml

```diff
@@ -11,4 +11,6 @@ data:
   rasaXToken: {{ .Values.rasax.token | b64enc | quote }}
   passwordSalt: {{ .Values.rasax.passwordSalt | b64enc | quote }}
   jwtSecret: {{ .Values.rasax.jwtSecret | b64enc | quote }}
+  initialUserName: {{ .Values.rasax.initialUser.username }}
```

Comment:
would you actually put this in the secret? I'm not sure 🤔

Reply:
yes! but it has to be base64-encoded as well